client/components/catalog/TagFilter.vue

```diff
@@ -5,40 +5,54 @@
         <v-icon color="primary lighten-4">mdi-tag-outline</v-icon>
       </v-btn>
     </template>
-    <v-list>
+    <v-sheet class="pa-4 primary darken-1">
```

Comment:
Dont forget to set tile attr 😉

Reply:
from some reason does not apply tile style

Reply:
You have outside border radius which is applied by v-menu component (setting radius on top corners of v-sheet and bottom for list; these ones are ok). But there is a border radius at the bottom of v-sheet (bordering with v-list). If you set tile attr that one will be gone.

![image](https://user-images.githubusercontent.com/6253820/77733199-be1e6880-7006-11ea-8c3b-4caadac86c59.png)